google-cloud-bigquery/lib/google/cloud/bigquery/table.rb

```diff
@@ -211,13 +211,12 @@ def time_partitioning_type
         #
         def time_partitioning_type= type
           reload! unless resource_full?
-          @gapi.time_partitioning ||=
-              Google::Apis::BigqueryV2::TimePartitioning.new
+          @gapi.time_partitioning ||= \
```

Comment:
This is really needed?

Reply:
Yes, all changes were made because of the updated style guide in the more recent Rubocop.